src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -118,9 +123,8 @@ contract SecurityCouncilNomineeElectionGovernor is
         _transferOwnership(params.owner);
 
         targetNomineeCount = params.targetNomineeCount;
-        firstCohort = params.firstCohort;
-        firstNominationStartTime = params.firstNominationStartTime;
-        nominationFrequency = params.nominationFrequency;
+        cohortOneFirstNominationStartDate = params.cohortOneFirstNominationStartDate;
```

Comment:
don't we only need one of these variables? i.e., just `firstElectionStartDate`, and then the yearsToAdd logic just adds 6 more months to that with each election?

Reply:
maybe, i just did two because of leap years and weird stuff like that.

Reply:
good point, not sure why i did this

Reply:
0af283f3e3107f24760972d424d5f5eac0df264a